relay_sidecar/elicitation.py

```
@@ -0,0 +1,110 @@
+    @staticmethod
+    def _clean_segment(value: str) -> str:
+        keep = [c if c.isalnum() or c in {"-", "_"} else "-" for c in value.strip()]
+        collapsed = "".join(keep)
+        return collapsed.strip("-")
+
+    def _decode_message(self, payload: dict[str, Any]) -> Message | None:
+        try:
+            meta = MessageMeta.model_validate(payload)
+            content = MessageContent.model_validate(payload.get("content", {}))
+            return Message(meta=meta, content=content)
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)